src/storage/accessors/FileDataAccessor.ts

```diff
@@ -70,6 +69,11 @@ export class FileDataAccessor implements DataAccessor {
     throw new NotFoundHttpError();
   }
 
+  public async* getChildren(identifier: ResourceIdentifier): AsyncIterableIterator<RepresentationMetadata> {
+    const link = await this.resourceMapper.mapUrlToFilePath(identifier);
```

Comment:
The name `link` is ambiguous to me, but perhaps we're already using it elsewhere?